src/main/java/com/mraof/minestuck/client/gui/playerStats/EcheladderScreen.java

```diff
@@ -332,27 +321,66 @@ public boolean mouseClicked(double xcor, double ycor, int mouseButton)
 		{
 			if(ycor >= yOffset + 35 && ycor < yOffset + 42)
 			{
-				scrollIndex = Mth.clamp(scrollIndex + RUNG_Y, 0, maxScroll);
-				isScrolling = true;
+				scroll = maxScroll;
```

Comment:
Click-based scrolling seems kinda off right now. It also looks to have changed from what it was before (though it did seem to be lacking before too, in the sense that only the up/down buttons were press-able, and not the rest of the scrollbar)

Reply:
Yeah, im not sure where it broke. This PR does nothing to change functionality and it remains the case that only the arrows can be clicked, which sends you to the very top or bottom. Otherwise you must use mouse scrolling

Reply:
I'll leave this to be solved separately then.